BVH.cs

```diff
@@ -79,6 +80,38 @@ public List<BVHNode<T>> Traverse(NodeTraversalTest hitTest)
 			return hits;
 		}
 
+		public SortedList<float, BVHNode<T>> SortedTraverse(NodeTraversalTest hitTest) {
```

Comment:
Can you please double check the curly braces are following the same convention across the project
As far as I know they should be on a newline